website/public/locale/en/translation.yaml

```diff
@@ -219,7 +218,7 @@ community:
     nextArrowAlt: Next event
     previousArrowAlt: Previous event
     noEvent:
-      message: No upcoming events!
+      message: CNCF hosted OpenEBS community events
```

Comment:
we can give this as cncf itself, noEvent cant be used here

Reply:
instead of `noEvent` key we can give `cncfEvents` or something like that

Reply:
since we display the cncf events only in the case when there are no events its fine!